README.md

```diff
@@ -13,7 +13,7 @@ High level process
 3. Run the new OpenShift ASP.NET base image
 
 For more information about using these images with OpenShift, please see
-the official OpenShift Documentation (https://docs.openshift.org/latest/using_images/s2i_images/php.html) .
+the official OpenShift Documentation (https://docs.openshift.com/enterprise/3.1/creating_images/s2i.html) .
```

Comment:
@sabre1041 Is it better to link to OSE or the Origin docs? hmm... The community docs might be more up-to-date https://docs.openshift.org/latest/creating_images/s2i.html